src/main/java/com/minecolonies/coremod/compatibility/resourcefulbees/ResourcefulBeesCompat.java

```diff
@@ -0,0 +1,39 @@
+package com.minecolonies.coremod.compatibility.resourcefulbees;
+
+import com.minecolonies.coremod.proxy.CommonProxy;
+import com.resourcefulbees.resourcefulbees.tileentity.TieredBeehiveTileEntity;
+import net.minecraft.client.Minecraft;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.RegistryKey;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.ModList;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Stack;
+
+public class ResourcefulBeesCompat

```

Comment:
this class and method should get some javadoc, else :shipit:

Reply:
will do in just a bit